sql/updates/1.7.1--1.7.2.sql

```diff
@@ -16,59 +16,66 @@ WITH
    -- All dimension slices that are mentioned in the chunk_constraint
    -- table but are missing from the dimension_slice table.
    missing_slices AS (
-      SELECT dimension_slice_id,
+      SELECT ch.hypertable_id,
+      	     di.id as dimension_id,
```

Comment:
Suggest fixing indentation (probably a tabs vs spaces issue).

Reply:
Ran untabify on the files.